### ISSUE TITLE
运行错误

### ISSUE THREAD

**Issue by huanghaozheyyds** [cqylunlun/GLASS#81]
回溯（最近一次调用最后）：
文件“/home/hhz/code/cv/GLASS/main.py”，第 354 行，在 main（）
文件中
“/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/click/core.py”，第 1157 行，在调用
中返回 self.main（*args， **kwargs）
文件“/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/click/core.py”，第 1078 行，在 main
rv = self.invoke（ctx）
中文件 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/click/core.py”，第 1720 行，调用
返回 _process_result（rv）
文件 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/click/core.py”，第 1657 行，_process_result
值 = ctx.invoke（self._result_callback， value， **ctx.params）
文件 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/click/core.py”， 第 783 行，在 invoke
return __callback（*args， **kwargs）
文件 “/home/hhz/code/cv/GLASS/main.py”，第 302 行，在运行
标志 = GLASS.trainer（dataloaders[“training”]， dataloaders[“testing”]， dataset_name）
文件 “/home/hhz/code/cv/GLASS/glass.py”，第 233 行，在 trainer 中
为 i，数据在 enumerate（training_data）：
文件 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py”， 第 630 行，在下一个
data = self._next_data（）
文件中 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py”，第 1345 行，
_next_data返回 self._process_data（data）
文件 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/utils/data/dataloader.py”，第 1371 行，_process_data
data.reraise（）
中文件“/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/_utils.py”，第 694 行，在 reraise
引发异常
FileNotFoundError：在 DataLoader 工作进程 0 中捕获 FileNotFoundError。
原始回溯（最近一次调用最后）：
文件“/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/utils/data/_utils/worker.py”，第 308 行，_worker_loop
data = fetcher.fetch（index）文件
“/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py”，第 51 行，在 fetch
data = [self.dataset[idx] for idx in possibly_batched_index] 中]
文件“/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py”，第 51 行，在
data = [self.dataset[idx] 中，对于 possibly_batched_index 中的 idx]文件
“/home/hhz/code/cv/GLASS/datasets/mvtec.py”，第 189 行，在 getitem
mask_fg = PIL 中。Image.open（fgmask_path）
文件 “/home/hhz/anaconda3/envs/glass_env/lib/python3.9/site-packages/PIL/Image.py”， 第 3247 行， in open
fp = builtins.open（filename， “rb”）
FileNotFoundError： [Errno 2] 没有这样的文件或目录： '/home/hhz/code/cv/GLASS/mvtec2d/fg_mask/bottle/150.png'
你好，我还想问一下我将mvtec数据集和dtd导进去后，run-mvtec.sh 中路径也修改好了，但是跑 run-mvtec.sh 时报出上述错误，如何解决呢，感谢你

**Comment by cqylunlun** [comment]
请下载foreground mask数据，详见[前景掩码](https://github.com/cqylunlun/GLASS?tab=readme-ov-file#4foreground-mask-download-link)节、问题 #68 和 #3 。

**Comment by huanghaozheyyds** [comment]
好嘞谢谢啦，祝您我都科研顺利
